src/player/audio.ts

```diff
@@ -144,12 +171,46 @@ export class AudioController {
           return
         }
         const elapsed = Date.now() - startTime
-        this.audio.volume = clamp(0.0, this.volume, from + (elapsed * step))
+        const baseVolume = clamp(0.0, this.volume, from + (elapsed * step))
+        this.audio.volume = baseVolume * replayGainFactor
         this.handle = setTimeout(run, 10)
```

Comment:
Btw. I don't think the fade will be correct now.. It may fade in too fast since replayGainFactor isn't induced in the step size